### PR TITLE
VTS documentation update

### PR DESCRIPTION
**Pull request by ines-toupeira**
- Rewrite and restructure the VTS documentation
- Enhanced clarity for Dominance, Reliability, and Availability factors
- Clarified the scope of VTS as focused on validator performance
The VTS documentation will be relocated once the new Validators section in the Developer Center is done.

### CODE REVIEW

File: learn/validator-trustscore.md [hunk 1]
```diff
@@ -66,66 +56,77 @@ $$
...
 
-Where $v$ is the validator's share and $Z$ is the total network share. This method applies when the epoch is active, so we can access each validator's balance using the `getActiveValidators` function from the RPC.
+Where:
+- $v$: The validator's share.
+- $Z$: The total network share.
 
-2. **Second method**: A different approach is used for a closed epoch. This is less accurate due to some randomness and is considered a fallback option. Here we look at the slot distribution of each voting block, which reflects the amount staked by each validator. The dominance ratio is calculated by dividing the number of slots allocated to a validator by the total number of slots in that epoch:
+This method relies on the `getActiveValidators` function from the [RPC](https://www.nimiq.com/developers/build/set-up-your-own-node/rpc-docs/#getactivevalidators), which provides real-time balances of each active validator.
+
+**Closed Epoch Method**:
```
Comment (onmax) [inline learn/validator-trustscore.md]:
```suggestion
**Finished Epoch Method**:
```

What do you think about using "finished" instead?

File: learn/validator-trustscore.md [hunk 2]
```diff
@@ -66,66 +56,77 @@ $$
...
+For a closed epoch, a less precise fallback method exists, but is not currently implemented. This approach calculates the dominance ratio based on slot distribution across voting blocks. The ratio is derived by dividing the number of slots allocated to a validator by the total slots in the epoch:
```
Comment (onmax) [inline learn/validator-trustscore.md:91]: Actually, it is implemented. In the code we have:
- `dominanceRatioViaBalance`: This value is exact. The only drawback is that we can only fetch this value when the epoch is active
- `dominanceRatioViaRatio`: This value is an approximation and not exact. This approximation is really close

Here is a screenshot of the values we have in our database:

image

Comment (onmax) [reply learn/validator-trustscore.md:91]: I can update this section of the docs if you don't have time or you are not sure. Let me know :)

File: learn/validator-trustscore.md [hunk 3]
```diff
@@ -136,130 +137,107 @@ Here you can see some examples depending on the stake percentage:
...
-
-$r_i$ is the number of blocks that the validator produced and received a reward for ($C_i$) divided by the number of blocks that the validator was likely to produce ($H_i$).
+The Reliability factor measures how consistently a validator produces blocks when expected. Validators who reliably produce their assigned blocks will have a high score, while those who fail frequently will receive a lower score. The reliability score is calculated as a weighted moving average over multiple epochs, emphasizing recent performance.
 
-</Callout>
-
-<details>
-
-<summary children:m-0>
+#### Calculating Reliability for an Epoch
```
Comment (onmax) [inline learn/validator-trustscore.md:144]:
```suggestion
#### Calculating Reliability for an Epoch

```

missing an empty line in between

File: learn/validator-trustscore.md [hunk 4]
```diff
@@ -1,59 +1,49 @@
 # Nimiq Validator Trustscore
 
-An algorithm that calculates a score to help users assess how reliable a validator is.{.subline .mb-32}
+<Callout type='warning'>
 
-The Validator Trustscore (VTS) algorithm is designed to help users assess the reliability of validators in the Nimiq Wallet. This score ranges from 0 to 1, where 0 indicates a validator is not trustworthy, and 1 indicates a highly trustworthy validator. This way, stakers can make better informed decisions about which validators to trust. The algorithm is based on three key factors:
+ The Validator Trustscore is still under heavy development and nothing is final. Feel free to share your [Suggestions and Feedback](#suggestions-feedback).
```
Comment (onmax) [inline learn/validator-trustscore.md]:
```suggestion
The Validator Trustscore is still under development and we are open to discuss changes to the algorithm.
```

The current copy feels like we are currently changing the algorithm, which is not the case. Maybe we should say that we are open to changing it if we think the feedback is useful?

**Review by onmax — APPROVED** [review]
I think this version is much more clear and readable. Thank you!

**Comment by onmax** [comment]
@Albermonte feel free to give your opinion, otherwise we will merge it beginning next week 🤘

File: learn/validator-trustscore.md [hunk 5]
```diff
@@ -1,59 +1,49 @@
...
-  <div flex="~ items-center gap-12" text-18 font-bold mb-8>
-    <div i-nimiq:bell text-16 op-70 />
-    Heads up
-  </div>
-  <p pr-32 md:pr-64>
-    The Validator Trustscore is still under heavy development and nothing is final. Feel free to share your <a href="#suggestions-feedback" underline text-inherit>Suggestions or Feedback</a>.
-  </p>
-  <div i-custom:crane absolute text-90 md:text-128 right--12 bottom--16 md:top-6 rotate-y-180 op-20 pointer-events-none />
-</div>
+The Validator Trustscore (VTS) helps users assess the reliability of validators in the Nimiq Wallet. This score ranges from 0 to 1, where 0 indicates a validator is not trustworthy, and 1 indicates a highly trustworthy validator. The VTS enables stakers make informed decisions when selecting validators.
```
Comment (Albermonte) [inline learn/validator-trustscore.md]:
```suggestion
The Validator Trustscore (VTS) helps users assess the reliability of validators in the Nimiq Wallet. This score ranges from 0 to 1, where 0 indicates a validator is not trustworthy, and 1 indicates a highly trustworthy validator. The VTS enables stakers to make informed decisions when selecting validators.
```

File: learn/validator-trustscore.md [hunk 6]
```diff
@@ -66,66 +56,77 @@ $$
...
+After determining the dominance ratio, a curve is applied to calculate the dominance score $S$:
 
 $$
 S = \max \left( 0 , 1 - s^{k} \right), \quad \text{being } t = 0.15 \text{ and } k=7.5
 $$
 
-Where $t$ is the threshold and $k$ is the slope of the curve.
+Where:
+
+- $t = 0.25$ is the threshold.
```
Comment (Albermonte) [inline learn/validator-trustscore.md]:
```suggestion
- $t = 0.15$ is the threshold.
```

File: learn/validator-trustscore.md [hunk 7]
```diff
@@ -66,66 +56,77 @@ $$
...
+- $k = 4$ is the slope of the curve.
```
Comment (Albermonte) [inline learn/validator-trustscore.md]:
```suggestion
- $k = 7.5$ is the slope of the curve.
```

File: learn/validator-trustscore.md [hunk 8]
```diff
@@ -136,130 +137,108 @@ Here you can see some examples depending on the stake percentage:
...
 
-We use the term _availability_ instead of _uptime_ because _uptime_ implies precise measurement, as in server contexts where you can measure online time. In our case, we can't measure how long a validator has been online. We can only see when validators are active and producing blocks. There's no way of telling when they're active but not producing blocks, or when they're offline.
+#### Why Availability Matters
 
-To be clear, a validator can be active and offline at the same time. It could be offline and not producing blocks because it's offline, or it could be active but not producing blocks because it hasn't been selected in a certain period. This is why we use _availability_ to show how often a validator is selected to produce blocks.
+Without availability, a validator could have high dominance and reliability scores but still not actively contribute to the network. This would misrepresent their role in maintaining the network's operation. The availability factor ensures only validators actively selected to produce blocks receive a higher score, penalizing those that are inactive, jailed, or offline.
```
Comment (Albermonte) [inline learn/validator-trustscore.md]: Here you are mentioning offline status, but in the next paragraph you say:
```
Availability focuses on participation in block production, regardless of online or offline status.
```

**Review by Albermonte — CHANGES_REQUESTED** [review]

File: learn/validator-trustscore.md (hunk 6 above)
Comment (ines-toupeira) [reply learn/validator-trustscore.md]: image
@onmax send me these updated values in the Hackathon
(?)

Comment (Albermonte) [reply learn/validator-trustscore.md]: This is the one: https://www.desmos.com/calculator/xynm6uphlq

**Review by Albermonte — APPROVED** [review]
LGTM

**Pull request merged**